src/main/java/net/snowflake/ingest/utils/Constants.java

```diff
@@ -74,6 +74,9 @@ public class Constants {
   public static final int PARQUET_MAJOR_VERSION = 1;
```

Comment:
Remove this hardcoded version and read the actual version from one of the parquet writers / parquet config classes ?

Reply:
Use major version from `ParquetFileWirter.CURRENT_VERSION`. Keep minor version as I cannot find one in `ParquetFileWriter`